API/src/core/ApiResponse.ts

```diff
@@ -0,0 +1,120 @@
+import { Response } from 'express';
+
+enum StatusCode {
```

Comment:
what's the use of StatusCode? 
I think for anything that isn't 200, the response code and message should be enough

Reply:
yeah, I think that status code is unnecessary actually.